regexes/device/mobiles.yml

```diff
@@ -562,6 +562,186 @@ Apple:
+      device: 'desktop'
+      model: 'MacBook Air 11" (2011)'
+
+    - regex: 'MacBookAir4(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2011)'
+
+    - regex: 'MacBookAir5(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2012)'
+
+    - regex: 'MacBookAir5(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2012)'
+
+    - regex: 'MacBookAir6(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2013)'
+
+    - regex: 'MacBookAir6(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2013)'
+
+    - regex: 'MacBookAir7(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2015)'
+
+    - regex: 'MacBookAir7(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2015)'
+
+    - regex: 'MacBookAir8(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2018)'
+
+    - regex: 'MacBookAir8(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2019)'
+
+    - regex: 'MacBookAir9(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2020)'
+
+    - regex: 'MacBookAir10(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2020)'
+
+    - regex: 'MacBookPro4(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro (2008)'
+
+    - regex: 'MacBookPro5(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2008)'
+
+    - regex: 'MacBookPro5(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2009)'
+
+    - regex: 'MacBookPro5(?:%2C|,)[34]'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2009)'
+
+    - regex: 'MacBookPro5(?:%2C|,)5'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2009)'
+
+    - regex: 'MacBookPro6(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2010)'
+
+    - regex: 'MacBookPro6(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2010)'
+
+    - regex: 'MacBookPro7(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2010)'
+
+    - regex: 'MacBookPro8(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2011)'
+
+    - regex: 'MacBookPro8(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2011)'
+
+    - regex: 'MacBookPro8(?:%2C|,)3'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2011)'
+
+    - regex: 'MacBookPro9(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2012)'
+
+    - regex: 'MacBookPro9(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2012)'
+
+    - regex: 'MacBookPro10(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2012)'
+
+    - regex: 'MacBookPro10(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2012)'
+
+    - regex: 'MacBookPro11(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2013)'
+
+    - regex: 'MacBookPro11(?:%2C|,)[23]'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2013)'
```

Comment:
According to https://support.apple.com/de-de/HT201300 those identifiers were used for models in 2013 and 2014. Seems that's also the case for some other models. Not sure if we should change the model name to `(2013/2014)` in such a case?

Reply:
If we specify a date range, we must accept this in the wiki agreement.

https://everymac.com/ultimate-mac-lookup/?search_keywords=MacBookPro11%2C3

2014-2015

Reply:
@sanchezzzhak we can do that for sure. Not sure how to handle that the best way....

Reply:
I think we need a few more opinions to make a decision based on a majority of yes votes.

Reply:
My opinion is that `2013/2014` can be used only for two consecutive years, while `2013-2015` can be used for multiple years.

Reply:
`2013-2015` is fine for me as well. Would you update the PR accordingly?